### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by vr8hub** [Gragt/charles-kingsley_the-water-babies#1]
- [x] We want elements to be semantically correct, e.g. tables should only be used for tabular data, blockquotes only for actual quotes, etc. IOW, we don't use an element just to get the formatting the element brings. For example, the contents of the tables here aren't tabular data, so something different should be used to accomplish the formatting. I've personally done two different things in different circumstances: one, use `display:table`, and two, use multiple columns. Either one is going to need a container, which can be a `<div>` (one of the rare instances when one might be OK), as it doesn't look like anything else is appropriate.
- [x] Ditto for the first blockquote in ch. 1 and the last in ch. 7; those look like unordered lists, not blockquotes, since there's nothing being quoted.
- [x] The item in the production notes re "divers" can be removed; modernize-spelling no longer makes that substitution (it was reverted late last year).
- [x] I don't think the quotes in chapter 6 (a6a4331) are necessary. It's quite common for quotes like that not to have quotemarks, I've seen it quite a bit over the years.
- [x] The Cu-ru-u-ick, cock-kick, etc. appear to be sounds the cock-grouse is making? If so, then the sounds should be unsemanticate italics (see [SEMoS 8.2.6](https://standardebooks.org/manual/1.7.3/single-page#8.2.6)). I think it would be just the sounds, not the words, e.g. come-<i>kick-kick-cock-kick</i>.
- [x] The smallcaps on Mrs Bedonebyasyoudid in ch. 5 don't seem to be strong emphases (i.e. yelling or close to it), but rather just formatting (especially since the second one is not in dialog). It would seem `<b>` is more appropriate?
- [x] (Nit) The "attemps" in the production notes should be "attempts".
- [x] The M. for Monsieur abbreviations should have a no-break-space between them and the name, just as Mr. and Mrs. do. (Typogrify handles it for Mr/Mrs and others, but since the M. are semanticated manually, the NBSP also has to be added manually.)
- [x] Only numbers 1 and 2 in the list in ch. 4 need a sub-list; 3/4/5 are at the top-level.
- [x] It might make the lists more readable to have them match the indentation of the scans, i.e. second and subsequent lines of an item indented? This can be accomplished with something similar to how the "iX" classes of poetry work.
- [x] Are you sure the three asterisks in the "Suffumigation" line is for obfuscation? I don't know it isn't (if there's a reference there I don't get it), it just seems like an odd use of spaced asterisks.
- [x] M-W has it as QED, the scans have it as Q.E.D, SEMoS doesn't special-case it, so shouldn't it be capitalized?
- [x] We only italicize poems that can stand alone, i.e. epic-length (see [SEMoS 8.2.11](https://standardebooks.org/manual/1.7.3/single-page#8.2.11)). _Bothie_ qualifies, “Ode to Duty” does not. You can still semanticate “Ode to Duty,” just with a span rather than italics.
- [x] When we have instances of a poem or part of a poem that have headers on the stanzas, it is another of the rare cases where we use a `<div>`, which then lets us use a `<header>` for the stanza header. See, e.g., Ameen Rihani's [_Poetry_](https://github.com/standardebooks/ameen-rihani_poetry/blob/0e3c5e49c19c6c2796ff18e747eca9140bf94a5b/src/epub/text/poetry.xhtml#L356). You can then target the headers in CSS.
- [x] Is there a reason you didn't match the small-caps of the "Come me my riddle…" paragraph in the dedication?

Several corner cases here. I'm _fairly_ confident in the following, but as always feel free to educate me in what I have missed or misunderstood. :) I'm going to ask Alex about at least one them myself, but I don't mind doing so for all of them.

- [x] The Balzac I just did had a couple of story-specific dedications, and he wanted them put in the file with the story rather than in a separate file. I believe he will want to do the same with these epigraphs, i.e. put them in the chapters as regular epigraphs, not in separate files. Among other things, we don't want them in the ToC. The question is whether they should be "normal" chapter epigraphs, or in their own section so they can, with CSS, still be on their own page. I don't have a strong opinion either way; to keep them on separate pages, the first section would be "chapter-x-epigraph" and the second "chapter-x-body" (which would be sections within the existing "chapter-x" section). See "Gobseck" in the Balzac Shorts from Scenes from Private Life for an example (there it's dedication, but the same concept would apply).
- [x] Normally lint would flag an se:name title that wasn't titlecased, e.g. "The History of the great and famous nation…", but the current check is only for <= 150 characters. Now, I don't know if that's because Alex doesn't care about titlecase for longer than, or if he's assuming anything longer than that really isn't a title. :) So, even though lint isn't complaining, all English titles normally have to be titlecased, whether they are in the source or not.
- [x] I know why you used cmn-Latn, I just don't know if it's correct (nor do I know that it _isn't_ correct). The only other instances we have of cmn in the corpus is cmn-Hant for Chinese names in _The Art of War_; for Chinese Latin characters I only see zh-Latn. This is one I'm truly ignorant on; if you have expertise let me know, otherwise one of our native Chinese speakers can probably speak to it.
- [x] All of the italicized words in ch. 4 don't feel like emphasis to me. If they are to be italicized, they would be unsemanticated italics, I think, but if there was only one or two they belong to the category of things we would normally scare quote. This is an example (weird italics) that trips me up a lot, so I'm just going to ask Alex about them.
- [x] As you well know, I'm not very good at long descriptions. This one is very interesting, but the middle three paragraphs are more about Kingsley than the book, and we (Alex) don't want a lot of detail about an author. I'm not suggesting changes, as I'm not qualified, just don't be surprised if Alex wants it shortened. Or, just as likely, he'll think it's the best thing since sliced bread. :)

**Comment by Gragt** [comment]
Still working on this—and learning a lot in the process!

Just one thing about the “suffumigation of the liver of the fish ***”: I interpreted the asterisks as obfuscating the name of the fish because the next line, which interrupts the list, goes: “Only they had forgotten its name, so Dr. Gray could not well procure them a specimen.” That said, if you still think the asterisks are correct, I’ll restore them.

**Comment by vr8hub** [comment]
Wouldn’t that confirm it _isn’t_ obfuscation—if they don’t know the name, there’s nothing to obfuscate. I would think an ellipsis would be more appropriate?

**Comment by Gragt** [comment]
Obscured is probably better than obfuscated here—sorry, bit tired. I mostly went with the description from [8.7.7.8](https://standardebooks.org/manual/1.7.3/single-page#8.7.7.8), specifically that it is the author’s decision not to divulge information—in this case because it is unknown to the characters.

What’s the final word then? Ellipsis, asterisks, or keep the three-em dash?

**Comment by vr8hub** [comment]
Same thing—em-dashes are used to obscure known things (town names, people names, etc.) that the author doesn't want known. This isn't a known thing, it's specifically stated as being _not_ known.
I don't think the asterisks are the thing to use, no matter what is going on, so I vote for the ellipsis.

**Comment by Gragt** [comment]
Ellipsis it is, then. Thanks!

**Comment by Gragt** [comment]
Alright, I think we are done here—unless I missed the obvious again!

I know you still need to check my changes, but let me thank you first for your patience and your clarity. There were a few things that I wasn’t sure about and even seemed incorrect, like the using the tables to format the text into columns, and I learned a lot from that—now I know we can actually use `<div>` for very specific reasons!

And then there were a few things that I knew but simply missed, like italicising the sounds—quite embarrassing, I must say. Good catch on all of those!

A few notes (which may or may not prompt you to ask me to change things one more time):

* The chapter 4 list uses a mix of `<ol>` and `<ul>`. It renders similar to the scans, with the items under 3, 4, and 5 at top-level as you asked. But I’m not sure if that’s semantically correct. If not, I may need some guidance on how to correct that.
* I had initially tried to get the longer list items to indent, specifically to improve readability, but couldn’t manage. I took another look at things and it turns out there was an element in `core.css` that I had to override specifically for this. It should indeed look nicer now.
* The second half of the dedication was not set in small caps because I thought it was more readable this way. That said, changing setting it to small caps was no biggie.
* I thought I had it right but did some research and it turns out I got it all wrong (of course!): *Fan Quei* is supposedly Cantonese and not Mandarin—how I managed that one is beyond me. I changed the tag to `yue-Latn` to reflect that but I’m not expert in Chinese. Also, it seems the correct way to write it in Latin script would be “fan gwei” but “fan quei” is close enough—I found a couple of sources for that (here is [one](https://en.wikipedia.org/wiki/Gweilo) and here is [another](https://ricepapermagazine.ca/2020/11/fan-gwei-lo-by-garry-engkent/)). Now, if you think that `zh-Latn` would be better, I can easily change that.
* I had a lenghty internal debate about the long description. I tend to get invested in the subject matter and do a lot of research. For what it’s worth, I’ve tried to stick to information that is directly relevant to the book and explains why the author wrote what he wrote. I guess we can let Alex figure it out and, if needed, I can rewrite it.
